generate-release/src/migration_guides.rs

```diff
@@ -127,7 +191,7 @@ fn generate_metadata_block(
     format!(
         r#"[[guides]]
 title = "{title}"
-url = "https://github.com/bevyengine/bevy/pull/{pr_number}"
+prs = [{pr_number},]
```

Comment:
```suggestion
prs = [{pr_number}]
```

Could we scrap the trailing comma since the array is in a single line?

```toml
prs = [123,]
# versus
prs = [123]
```